src/web/components/Figure.tsx

```diff
@@ -8,6 +8,7 @@ type Props = {
 	role?: RoleType;
 	children: JSXElements;
 	id?: string;
+	dataCy?: string;
```

Comment:
I'm hesitant to add props for test purposes. I find that there's a finite number of pops that you can have before code gets hard to reason about and brittle and because of this adding new ones comes with a cost and I'm not sure we want to make that spend just for Cypress?

Reply:
I think if we need an element to have a data attribute then we should just set it manually. Shall we just add this `data-cy` attribute to the component directly?